portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,33 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+        body {
+            background-image: url('https://i.gifer.com/GHSe.gif');
+            }
+    </style>
+    <link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope|Nanum+Pen+Script&display=swap" rel="stylesheet">
   </head>
   <body>
+      <div class="nav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+  <a href="#contact">Contact</a>
+  <a href="about.html">About Me</a>
+  </div>
     <div id="content">
+    <div style="background-color:white;background-image:url(//www.html.am/images/backgrounds/background-image-2.gif);background-repeat:repeat;"></div>
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <br>
+      <p>This is my domain!~ Welcome :) In this webpage, I will be explaining a bit more about myself. I hope you enjoy your stay!</p>
       <div id="greeting-container"></div>
```

Comment:
Is this greeting-container needed? I don't think it is really doing anything right?

Reply:
It was there by default! I forgot to remove it, haha.